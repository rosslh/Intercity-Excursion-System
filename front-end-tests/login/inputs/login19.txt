login planner
changeticket 12345 54321 10
logout