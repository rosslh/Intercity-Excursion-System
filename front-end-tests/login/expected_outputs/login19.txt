CHG 12345 54321 10
EOS
